.github/dependabot.yml

```diff
@@ -0,0 +1,53 @@
+# https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+
+updates:
+  - commit-message:
```

Comment:
I don't think we want this running on the main composer.json. This is not an app, and we can can keep track of what requirements we want in there ourselves.